plugins/explore/src/components/ToolCard/ToolCard.test.js

```diff
@@ -14,11 +14,10 @@
  * limitations under the License.
  */
 
-import React from 'react';
```

Comment:
This file seems to be us `.js` vs `.ts` for some reason

Reply:
Yeah, this seems to be a very old file 😆 Not sure if we want to fix that, just changing the extension is not enough. I will have a look how much work it is.

Reply:
Works!